### PR TITLE
Enable notifications/integrations on ci-stable

### PR DESCRIPTION
**Pull request by josejulio**
I'll open a similar PR for `qa-stable` and `stage-stable` once this one is merged

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -379,6 +379,22 @@ insights:
...
+  api:
+    versions:
+      - v1
+  frontend:
+    app_base: notifications
+    module:
+      appName: integrations
+      scope: notifications
+      module: ./RootApp
+      manifest: /apps/notifications/fed-mods.json
```
Comment (josejulio) [inline main.yml:394]: Should I remove this one on stable? `validate-yaml` is complaining

File: main.yml [hunk 2]
```diff
@@ -669,12 +708,28 @@ user-preferences:
...
       - /user-preferences
     sub_apps:
       - id: email
         title: Email preferences
         default: true
+        group: email
+      - id: notifications
+        title: Notifications
+        group: email
+        sub_apps:
```
Comment (josejulio) [inline main.yml:726]: After removing the `manifest` (locally) it's complaining about this one:
```
Error in user-preferences app definition.
 [Error [ValidationError]: "frontend.sub_apps[1].sub_apps" is not allowed] {
  _original: {
    title: 'User Preferences',
    deployment_repo: 'https://github.com/RedHatInsights/user-preferences-frontend-build',
    top_level: true,
    frontend: {
      title: 'User preferences',
      module: [Object],
      paths: [Array],
      sub_apps: [Array]
    },
    source_repo: 'https://github.com/RedHatInsights/user-preferences-frontend'
  },
  details: [
    {
      message: '"frontend.sub_apps[1].sub_apps" is not allowed',
      path: [Array],
      type: 'object.unknown',
      context: [Object]
    }
  ]
}
```

File: main.yml (hunk 1 above)
Comment (karelhala) [reply main.yml:394]: We didn't updated the yaml validation on stable branch as of yet - this PR should solve your problem https://github.com/RedHatInsights/cloud-services-config/pull/522

File: main.yml (hunk 2 above)
Comment (karelhala) [reply main.yml:726]: Yeah, the validator is not up to date on stable branch.

**Review by karelhala — APPROVED** [review]
Looking good!

**Comment by karelhala** [comment]
Restarting travis

**Comment by josejulio** [comment]
Travis is happy now, thanks!

**Pull request merged**